rust/cubestore/cubestore/src/metastore/rocks_store.rs

```diff
@@ -675,53 +675,51 @@ impl RocksStore {
         let rw_loop_sender = self.rw_loop_tx.clone();
         let (tx, rx) = oneshot::channel::<Result<(R, Vec<MetaStoreEvent>), CubeError>>();
 
-        cube_ext::spawn_blocking(move || {
-            let res = rw_loop_sender.send(Box::new(move || {
-                let db_span = warn_long("store write operation", Duration::from_millis(100));
-
-                let mut batch = BatchPipe::new(db_to_send.as_ref());
-                let snapshot = db_to_send.snapshot();
-                let res = f(
-                    DbTableRef {
-                        db: db_to_send.as_ref(),
-                        snapshot: &snapshot,
-                        mem_seq,
-                        start_time: Utc::now(),
-                    },
-                    &mut batch,
-                );
-                match res {
-                    Ok(res) => {
-                        if batch.invalidate_tables_cache {
-                            *cached_tables.lock().unwrap() = None;
-                        }
-                        let write_result = batch.batch_write_rows()?;
-                        tx.send(Ok((res, write_result))).map_err(|_| {
-                            CubeError::internal(format!(
-                                "[{}] Write operation result receiver has been dropped",
-                                store_name
-                            ))
-                        })?;
-                    }
-                    Err(e) => {
-                        tx.send(Err(e)).map_err(|_| {
-                            CubeError::internal(format!(
-                                "[{}] Write operation result receiver has been dropped",
-                                store_name
-                            ))
-                        })?;
+        let res = rw_loop_sender.send(Box::new(move || {
```

Comment:
Spawn blocking doesn't provide any safety here. Sending in mpsc channel is an async operation. What do we need it?